Makefile

```diff
@@ -93,17 +93,22 @@ cmd/security-bootstrapper/security-bootstrapper:
 clean:
 	rm -f $(MICROSERVICES)
 
-hadolint:
-	if which hadolint > /dev/null ; then hadolint --config .hadolint.yml `find * -type f -name 'Dockerfile*' -print` ; elif test "${ARCH}" = "x86_64" && which docker > /dev/null ; then docker run --rm -v `pwd`:/host:ro,z --entrypoint /bin/hadolint hadolint/hadolint:latest --config /host/.hadolint.yml `find * -type f -name 'Dockerfile*' | xargs -i echo '/host/{}'` ; fi
-	
-test: hadolint
+unittest:
 	go mod tidy
 	GO111MODULE=on go test $(GOTESTFLAGS) -coverprofile=coverage.out ./...
 	GO111MODULE=on go vet ./...
 	gofmt -l .
 	[ "`gofmt -l .`" = "" ]
 	./bin/test-attribution-txt.sh
```

Comment:
These should be moved under `test` target since that are no `unit tests`. Do we need `go vet` if using linter? Never seen it complain about anything...

Reply:
https://sparkbox.com/foundry/go_vet_gofmt_golint_to_code_check_in_Go

Suggests that go vet is relatively important not to ignore and that lint is more subjective.

Reply:
Please see if new arrangement works better.

Reply:
Yep that works. THX!